openfermionpyscf/tests/_run_pyscf_test.py

```diff
@@ -0,0 +1,43 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Thanks for adding a test! If this package is going to get built out a bit more it is going to be necessary to setup continuous integration (automatic testing). However, to do that, we need pyscf to be easy enough to install that it can install automatically using pypi on Travis. This is not necessary for this PR, I will open an issue about it.